frameit/lib/frameit/runner.rb

```diff
@@ -82,8 +82,12 @@ def editor(screenshot, config)
     # Loads the config (colors, background, texts, etc.)
     # Don't use this method to access the actual text and use `fetch_texts` instead
     def create_config(screenshot_path)
+      # Screengrab pulls screenshots to a different folder location
+      # frameit only handles two levels of folders, to not break
+      # compatibility with Supply we look into a different path for Android
+      # Issue https://github.com/fastlane/fastlane/issues/16289
       config_path = File.join(File.expand_path("..", screenshot_path), "Framefile.json")
-      config_path = File.join(File.expand_path("../..", screenshot_path), "Framefile.json") unless File.exist?(config_path)
+      config_path = File.join(File.expand_path("../../../..", screenshot_path), "Framefile.json") unless File.exist?(config_path)
```

Comment:
Do we want to keep the `../..`? I don’t know if removing that is actually breaking or not but it might be safer to keep it in 🤔 Thoughts?

Reply:
Hey @joshdholtz, I was waiting for someone to reply in here, I guess it makes sense so we don't break compatibility with an older path.
I'll put it back.  thanks for replying.